### PR TITLE
API Tokens, Allow a User Creates a New API Token

### LINKED ISSUE #42: Add button create API Token
We have to add a button with the goal the user has the ability to create an API token as needed.

### PR DESCRIPTION
**Pull request by wzalazar**
This PR introduces

- Adds the button ``` Get API Token ``` to the la UI
- Improves code in the container
- Fixes the height of the box of tokens allows showing the 5 tokens
- Fixes actions names, and filenames
- Updates Frost Client to 0.1.4

fix #42

### CODE REVIEW

File: src/components/atoms/NotificationBar/NotificationBar.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as classNames from 'classnames'
+import * as React from 'react'
+import './NotificationBar.scss'
+
+interface NotificationBarProps {
```
Comment (lautarodragan) [inline src/components/atoms/NotificationBar/NotificationBar.tsx]: This is very similar to NotificationBar from src/interfaces/Props.ts.

Accidental duplication?

File: src/reducers/NotificationBar.reducer.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        ...state,
+        action: 'fade-out'
+      }
+
+    case Actions.NotificationBar.RESET_NOTIFICATION_BAR:
+      return {
+        ...defaultState
+      }
+  }
+  return state || defaultState
```
[the remark this answers is not in the document]
Comment (wzalazar) [reply src/reducers/NotificationBar.reducer.ts]: not, it's equal. I will change it.

[the remark this answers is not in the document]
Comment (lautarodragan) [reply src/reducers/NotificationBar.reducer.ts]: Either way is fine, just wanted to know

File: src/components/atoms/NotificationBar/NotificationBar.tsx (hunk 1 above)
Comment (lautarodragan) [reply src/components/atoms/NotificationBar/NotificationBar.tsx]: Got it. So redux state interfaces are in interfaces/Props, but Props interfaces are in the same file as the components?

File: src/reducers/NotificationBar.reducer.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (wzalazar) [reply src/reducers/NotificationBar.reducer.ts]: ok

File: src/components/atoms/NotificationBar/NotificationBar.tsx (hunk 1 above)
Comment (wzalazar) [reply src/components/atoms/NotificationBar/NotificationBar.tsx]: yes

**Pull request merged**